### ISSUE TITLE
Nier name input is broken after #5465

### ISSUE THREAD

**Issue by 13xforever** [RPCS3/rpcs3#5494]
After implementing OSK syscalls in #5465 Nier now refuses any name input data when starting a new game.

[nier_7645_works.zip](https://github.com/RPCS3/rpcs3/files/2723480/nier_7645_works.zip)
[nier_7650_doesn't_work.zip](https://github.com/RPCS3/rpcs3/files/2723482/nier_7650_doesn.t_work.zip)

**Comment by lowsisad** [comment]
Hey! I have this problem to!
I have download and use several versions, but i cant input name.
Nor in japanese, not in english. I copied words from google/translation , but it doesnt work.
Only saves from other players.

**Comment by nuxastas** [comment]
i have the same problem ,i tried with nier gestalt and replicant both of them have the same problem.
Anyone have a savefiles just in the begining of the game?

**Comment by MSuih** [comment]
Test with #5497

**Comment by lowsisad** [comment]
> Test with #5497

Could you say, now to download this branch?

**Comment by MSuih** [comment]
If you are on windows:
1. Open the commits tab on the pull request.
2. Click the bottommost green check mark (✅ symbol).
3. Click on Details link for appveyor.
4. Open artifacts tab in appveyor and download the largest file.

**Comment by Kaeril** [comment]
I confirm #5497 fixes it!

**Comment by lowsisad** [comment]
No, it doesnt work.
I have put random simbols from this https://www.lexilogos.com/keyboard/hiragana.htm and i had invalid name again.

**Comment by Kaeril** [comment]
![sans titre](https://user-images.githubusercontent.com/2845458/50727413-bf6e2780-111a-11e9-8887-f09c9d5bb770.png)
I'm sorry to insist, but that build made it work fine here. I tested this from a fresh install, wiped out the rpcs3 folder just to make sure. I'm not too familiar with rpcs3 yet, but did you try recompiling the PPU modules?

**Comment by lowsisad** [comment]
> I'm sorry to insist, but that build made it work fine here. I tested this from a fresh install, wiped out the rpcs3 folder just to make sure. I'm not too familiar with rpcs3 yet, but did you try recompiling the PPU modules?

Could you give me a link on your copie of the game? Where had you have to download it?
Maybe it is a source of problems.

**Comment by RainbowCookie32** [comment]
@lowsisad https://store.playstation.com/en-us/product/UP0082-BLUS30481_00-NIER000000000000

**Comment by lowsisad** [comment]
> @lowsisad https://store.playstation.com/en-us/product/UP0082-BLUS30481_00-NIER000000000000

Are you buy Nier at the shop and after are playing on PC?

**Comment by Megamouse** [comment]
Then dont use hiragana :/

**Comment by lowsisad** [comment]
> Then dont use hiragana :/

I tried english. Nothing.
I'm not sure: this emulator can works with pirates copies?

**Comment by RainbowCookie32** [comment]
> 
> 
> > Then dont use hiragana :/
> 
> I tried english. Nothing.
> I'm not sure: this emulator can works with pirates copies?

It can work, but we don't support piracy

**Comment by RainbowCookie32** [comment]
> 
> 
> > Unrelated to this issue
> > I mean, mb it's my mistake, bc im trying to input name from the real keyboard.
> > http://prntscr.com/m3g5pk

No, that prompt is for you to input from your keyboard, you are not doing it wrong

**Comment by lowsisad** [comment]
Mb i need to change something there or in another place?

http://prntscr.com/m3ggew
http://prntscr.com/m3ggos

**Comment by RainbowCookie32** [comment]
> 
> 
> Mb i need to change something there or in another place?
> 
> http://prntscr.com/m3ggew
> http://prntscr.com/m3ggos
> 
> > > > Unrelated to this issue
> > > > I mean, mb it's my mistake, bc im trying to input name from the real keyboard.
> > > > http://prntscr.com/m3g5pk
> > 
> > 
> > No, that prompt is for you to input from your keyboard, you are not doing it wrong
> 
> And say, emulator dont save some conf in the system? Bc i havent delete old version of emulator.

You are having problems that the other user is not having, you have a pirated copy. It sounds more like you pirated copy is broken

**Comment by lowsisad** [comment]
> You are having problems that the other user is not having, you have a pirated copy. It sounds more like you pirated copy is probably broken

![image](https://user-images.githubusercontent.com/42754481/50728682-e50d2800-113e-11e9-85e2-a979fb7dda69.png)
I have tried with 2 diffrent versions of Nier and with Demon's Souls. In Demon's Souls i haven't met this error.

Ok, i'll wait the answer of 13xforever . If he dont have error too, ill calm down.

**Comment by Megamouse** [comment]
Please dont post here if you have no idea what you are doing

**Comment by lowsisad** [comment]
> Please dont post here if you have no idea what you are doing

Please, don't be so rude. I know what i'm doing.

**Comment by AniLeo** [comment]
No, you don't, you are spamming this while using pirated copies

**Comment by Retr0Fissi0n** [comment]
I've installed the #5497 build and while it now allows me to input a name, the game crashes immediately after doing so, without asking me to confirm if this was the name I wanted to use. I have practically no experience with RPCS3 whatsoever and frankly I only installed it to play Nier. The configurations for CPU, GPU etc. haven't been changed and are on their default settings at all, so I'm wondering if I perhaps need to change any of those or if I'm having a different problem entirely.

**Comment by Cerv4ntez** [comment]
I struggled through the naming problem all day and managed to enter the name with build 7645 as mentioned in the initial post here. This is currently broken again, since native input has been implemented and there is no way to switch to japanese characters. This only concernes the japanese variety nier replicant of course.

**Comment by Megamouse** [comment]
you would only have to disable the native ui. no need to use an old build.
As to using japanese characters... that's probably not gonna happen in a while

**Comment by elemileTLDR** [comment]
Just saying, those who had to input a japanese name (like me) will need to disable the native ui in Emulator settings by the end of the game, since you have to input your name again and there's no other way than pasting the text from some text file (ctrl+v and enter). I tried changing both yhe console and keyboard language to japanese, but the game ui still only displays latin characters (that didn't let you use in the first place).
